packages/widgets/src/components/bubble/bubble-chart.widget.tsx

```diff
@@ -25,6 +36,17 @@ export const BubbleChartWidget: FC<Props> = ({
   card,
   ...props
 }) => {
+  const { data, valueKey } = props;
+  const values = getValues(data, [valueKey]);
```

Comment:
Could be replaced with `calculateRange` function from `@keen.io/charts -> data.utils`;